Add trilateral datasets

danielfdsilva opened the pull request:
Include the datasets highlighted in the [spreadsheet](https://docs.google.com/spreadsheets/d/18VwfBD4h0WWVlZR93tzLhmXJDZH8aDh-O71ZsRMMKTc/edit#gid=773624342).

Related issue: https://github.com/NASA-IMPACT/delta-config/issues/85

Summary:
- All the datasets were included with very limited information.
- Most of them show up on the map all black. I'm assuming it is because some `source_params` are missing.
- Images are missing for all datasets. Showing a placeholder.
- Legend is missing on all datasets.
- The titles and descriptions need to be reviewed. For some they're too long (like `geoglam`) and for others too cryptic.
- Content about the dataset is also needed (long form about page)
- Created a new Biomass thematic area. Description and content needed.
- The items `grdi-v1-built` and `grdi-v1-raster` seemed related so I added them as layers under the same dataset (`grdi-v1`). Same for `grdi-vnl-raster` and `grdi-vnl-slope-raster`.

aboydnw commented:
Thanks Daniel. A couple questions on your summary above. 

> - Most of them show up on the map all black. I'm assuming it is because some source_params are missing.

What are these source_params? Who can we work with to get these working? cc @michaelgsuttles 

> Images are missing for all datasets. Showing a placeholder.

I can send this request to IMPACT.

> - Legend is missing on all datasets.
> - The titles and descriptions need to be reviewed. For some they're too long (like geoglam) and for others too cryptic.
> - Content about the dataset is also needed (long form about page)

Can we use the links I provided in the [spreadsheet](https://docs.google.com/spreadsheets/d/18VwfBD4h0WWVlZR93tzLhmXJDZH8aDh-O71ZsRMMKTc/edit#gid=773624342) for the long form content and legends? Also, when it makes sense, maybe we could use the titles in the menu for titles. I highlighted these below.

![image](https://user-images.githubusercontent.com/92742765/187720422-bda45f4e-6233-47b0-ac36-cfb14d7a6957.png)

danielfdsilva commented:
The `source_params` are any params necessary for the tiler. Things like `bidx, colormap_name, rescale`. Perhaps @anayeaye can help?

Images - It would be nice to have pretty photos (a field for crops maybe), and not a graph or something with text.

I'll look at the links for information.
For the legend it would be good to get the stops colors or the colorscale name, just so we don't need to eyedrop the color

danielfdsilva commented:
Quick update:
- Added the about content and legend for geoglam.
- Merged all GRDI under one dataset since in the website they all had the same about content. This makes more sense hoestly. Added about content and legend.

Datasets are still not correctly visible on the map

anayeaye commented:
@aboydnw @danielfdsilva I should have exported the collection summaries for that table. 

My thoughts are that we can set some minimal source params to see something on the map and then ask scientists for colormap suggestions after they see everything in black and white?
- get ballpark rescale values from the collection summaries by clicking through on the stac records url (collection>summaries>default_cog)
- use colormap_name=gray and for the legend
- ```legend:
      type: gradient
      min: "low"
      max: "high"
      stops:
        - "#000000"
        - "#FFFFFF"
```

I just dropped in midstream here, though, so I apologize if I'm not getting to the question

anayeaye commented:
@danielfdsilva I had a look at the covid-api to get a start on the geoglam display. It looks like we can make it work with the path parameter `colormap`. I also think this is probably something we could encode in the backend to make it easier but for now I think it will work. 

I think we can just add this `colormap`  to the sourceParams: `colormap={"1": [120, 120, 120, 255], "2": [130, 65, 0, 255], "3": [66, 207, 56, 255], "4": [245, 239, 0, 255], "5": [241, 89, 32, 255], "6": [168, 0, 0, 255], "7": [0, 143, 201, 255]}`

References:
legend: https://github.com/NASA-IMPACT/covid-api/blob/develop/covid_api/db/static/datasets/agriculture.json#L39
custom crop monitor https://github.com/NASA-IMPACT/covid-api/blob/master/covid_api/api/utils.py#L512:L520
titiler-pgstac: https://stac-utils.github.io/titiler-pgstac/mosaic_endpoints/

danielfdsilva commented:
@anayeaye Cool! This seems to be working. I also added the other params that were present on your example preview, just for good measure.

<img width="975" alt="image" src="https://user-images.githubusercontent.com/1090606/188437207-96d00eb9-0ad1-4db7-a863-8adf4f9ac3df.png">

danielfdsilva commented:
@aboydnw Included the Biomass about content and the biomass story (#101).
I used the first part of the document as the about info for the biomass as it seemed separate from the rest. Let me know if this isn't the case.

Some titles and descriptions are still needed.

aboydnw reviewed (approved):
Looks good to me. I say we call this chunk done and send out for feedback.
